src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -36,9 +36,6 @@ public void testIsFull() {
 			Assert.assertEquals(false, result);
 			rhymer.countIn(888);
 		}
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project.